mle/agents/coder.py

```diff
@@ -73,6 +75,8 @@ def __init__(self, model, working_dir='.', console=None, single_file=False):
             - You should create a single script first, with the complete code inside. You can have multiple functions and classes.
             - Writing clean, efficient, and well-documented code to a script using functions `create_file`.
             - Use function `preview_csv_data` to preview the CSV data if the task include CSV dataset or examples.
+            - Use function `preview_zip_structure` to preview the structure of the file if the task include zip file processing.
```

Comment:
It seems that we are continuously improving our prompts. now its time to use some prompt tracking tools to help us do some A/B testings